Patch to add Solaris to the list of supported OSes.  This patch also
removes dependencies on other Puppet modules which do not apply to
Solaris.  The first piece of this patch has not yet been submitted
upstream but the second piece is not suitable for upstream
contribution.

--- saz-memcached-2.7.1/metadata.json.orig	2015-04-23 15:25:39.614824153 -0600
+++ saz-memcached-2.7.1/metadata.json	2015-04-23 15:26:36.049638360 -0600
@@ -28,6 +28,9 @@
     },
     {
       "operatingsystem": "Windows"
+    },
+    {
+      "operatingsystem": "Solaris"
     }
   ],
   "requirements": [
@@ -42,7 +45,6 @@
   ],
   "description": "Manage memcached via Puppet",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 3.2.0"},
-    {"name":"puppetlabs/firewall","version_requirement":">= 0.1.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">= 3.2.0"}
   ]
 }

--- saz-memcached-2.8.1/README.md.orig	2016-04-27 12:23:14.628880726 -0700
+++ saz-memcached-2.8.1/README.md	2016-04-27 12:23:27.017435303 -0700
@@ -1,3 +1,5 @@
+NOTE: Oracle has modified the originally distributed contents of metadata.json.
+
 # puppet-memcached [![Build Status](https://secure.travis-ci.org/saz/puppet-memcached.png)](http://travis-ci.org/saz/puppet-memcached)
 
 Manage memcached via Puppet
